wadsrc/static/zscript/mapdata.zs

```diff
@@ -212,6 +212,55 @@ struct SecPlane native play
 	native double PointToDist(Vector2 xy, double z) const;
 }
 
+struct F3DFloor native play
+{
+	enum EFFloorType
```

Comment:
I would change it to something like `F3DFloorFlags` as it doesn't represent a floor type but combination of its properties. This type name shouldn't be used directly though.

Reply:
Done